packages/react-components/lib/robots/robot-info.tsx

```diff
@@ -15,6 +15,20 @@ function getTaskStatusDisplay(assignedTask?: string, taskStatus?: string) {
   }
 }
 
+function calculateLastUpdateTime(
+  updateTime: number,
+  lastUpdateTime: number,
+  timeOfLastUpdate: number,
+): string {
+  if (lastUpdateTime === updateTime) {
+    return ((Date.now() - timeOfLastUpdate) / 1000).toFixed(1);
+  } else {
+    lastUpdateTime = updateTime;
+    timeOfLastUpdate = Date.now();
+    return '< 1';
+  }
```

Comment:
With this, I was facing the issue of the latest update keeps showing "<1", despite the fleet_states are not updating from rmf.

It works after changing this  function block impl and changing it to a single line
```
  return ((Date.now() - updateTime) / 1000).toFixed(1);
```
I believe this is actually sufficient and we don't need `lastUpdateTime` and `timeOfLastUpdate` argument.

Reply:
@mayman99   This feature with the above changes is tested and proved well in https://github.com/open-rmf/rmf-web/pull/603/files#r903270019.

Understand that this might be incorrect for `use_sim_time` sim scenario. One way to move forward is to merge this helpful feature in to `main`, while opening an issue to acknowledge this sim_time issue when running it on simulation.

Reply:
Sorry I just saw your comment right now, I agree :+1: